app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class AuditMailer < ApplicationMailer
+  def failure(audit: nil, package:, error:)
```

Comment:
The `audit.user.email` call below will raise a NoMethodError with this default.  This default seems to be here because `FixityCheckJob#perform` allows audit to be nil.  `FixityCheckJob` only has that support so that it can be used to do slightly different things in `AuditsController` and `PackagesController`.  We should decompose `FixityCheckJob` into two classes that exactly address the needs of the two callers, then look for opportunities for abstraction.